model/src/main/proto/spine/validation/rule.proto

```diff
@@ -132,13 +132,24 @@ message CompositeRule {
     // runtime with the error messages of the `left` and `right` rules and the words "and" or "or"
     // or "xor".
     //
+    // May also include the token `{value}`, but only if the `common_field` is also set.
+    //
     string error_message = 4;
+
+    // If this rule describes several constraints of the same field, this is the name of the field.
+    //
+    // It is only used for diagnostic purposes. The constraints themselves are always validated
+    // according to the rules `left` and `right`.
+    //
+    // If `left` and `right` do not describe constraints of the same field, this value is empty.
+    //
+    protodata.FieldName common_field = 5;
```

Comment:
I would never guess what this field is by its name.

Can we rename it to `constrained_field`? With the same docs.

Reply:
Renamed to `field`, just like in `SimpleRule`.

The intention was to show that the field is common across `left` and `right` rules. But, if the name confuses the reader rather than helps, I'd prefer a shorter name.